setup.py

```diff
@@ -62,7 +62,7 @@
         "requests",
         "rich>=10",
         "rich-click",
-        "spectra>=0.0.10",
+        "spectra>=0.0.10"
```

Comment:
The Black code linter will complain with out this..

```suggestion
        "spectra>=0.0.10",
```

I recommend setting up [pre-commit](https://pre-commit.com/) to run these tests automatically before pushing.

```bash
pip install pre-commit
# Set up MultiQC repo specifically, run from repo root dir
pre-commit install
# After that, pre-commit will run tests every time you do:
git commit -m "My changes"
# If anything fails (eg. a comma breaking the linting) it'll hopefully be fixed automatically, but it'll cancel the commit. So you need to then do:
git add .
git commit -m "My changes"
#.. then the pre-commit tests should pass.
```